### ISSUE TITLE
Bug: Export CSV does not export comments, or content

### ISSUE THREAD

**Issue by sbishel** [mattermost/focalboard#1002]
**Summary:**
Again, probably known and by design.
But Export CSV does not export either comments, or content

**Steps to reproduce the behavior:**
1. Go to board, created cards with comments and content
2. Export board as csv.
3. Look at csv file
4. Neither comments, nor content appear in CSV.

**Expected behavior:**
I would expect at the comments and content as text to be exported

**Comment by jespino** [comment]
This is designed that way, it only exports the visible options on the view, that means for tables, is a "what you see is what you get", and for kanban the same.

@chenilim can have a better understanding of what the intention is there.

**Comment by chenilim** [comment]
Yes, agree with Jesus - exporting a table to a CSV should generally export what is shown in the view. However, exporting a board view to a CSV is a bit more ambiguous. That said, I can see a need to export content and comments for some use cases.

I can think of a couple long-term approaches:
1. Allow content and comments to be shown in tables, and thus exported as part of the CSV
* The display would be weird, as they would generally be multi-line within a cell
2. Or, add an export to CSV dialog that adds options to include content and / or comments

Either way, the content and comments would be encoded as a single cell in the CSV.

Right now, my sense is these use-cases are P3 (nice-to-have), but open to input from others if you have a stronger need for this. cc @asaadmahmood.

**Comment by pullopen** [comment]
> 
> 
> Yes, agree with Jesus - exporting a table to a CSV should generally export what is shown in the view. However, exporting a board view to a CSV is a bit more ambiguous. That said, I can see a need to export content and comments for some use cases.
> 
> I can think of a couple long-term approaches:
> 
>     1. Allow content and comments to be shown in tables, and thus exported as part of the CSV
> 
> 
>     * The display would be weird, as they would generally be multi-line within a cell
> 
> 
>     1. Or, add an export to CSV dialog that adds options to include content and / or comments
> 
> 
> Either way, the content and comments would be encoded as a single cell in the CSV.
> 
> Also note that you can export the jsonl archive in order to get a record of the content and comments.
> 
> Right now, my sense is these use-cases are P3 (nice-to-have), but open to input from others if you have a stronger need for this. cc @asaadmahmood.

In Notion, they export a `.zip` including csv and Markdown files of the detailed pages. I think that would also be a good choice.

**Comment by asaadmahmood** [comment]
Yup, I think definitely a good to have, maybe these can actually be options within the export dropdown/modal to allow users to  customise the export.
